### PR TITLE
Tra nl nl

### PR DESCRIPTION
**Pull request by rvdb**
pkp/pkp-lib#4503 updated nl_NL locale

### CODE REVIEW

File: locale/nl_NL/admin.xml [hunk 1]
```diff
@@ -11,54 +11,69 @@
   * Localization strings for the nl_NL (Nederlands) locale.
   *
   -->
-
 <locale name="nl_NL" full_name="Nederlands">
-	<message key="admin.hostedJournals">Tijdschriften</message>
```
Comment (mtub) [inline locale/nl_NL/admin.xml]: Here in throughout in all files: please fix indentation - one tab, no spaces

**Comment by mtub** [comment]
I can merge this into `master` and cherry-pick to `stable-3_1_2` for the next release. If you intend to use this translation for OJS 3.1.1 as well, you'll probably have to look again at the keys you have deleted. But providing this for the most recent branches is more important, I think.

**Comment by rvdb** [comment]
@mtub Ah, sorry for the tabs; that's the downside of comparing with XML-aware tools that don't care much about whitespace. I'll fix this, but maybe I should address the ojs-stable-3_1_1 issue as well. I realize I've now aligned my translations to the master branch. In order to make my effort as broadly applicable as possible, what is the best approach? Should I align the translations to the ojs-stable-3_1_1 branch instead, thus removing keys that are in master but not in ojs-stable-3_1_1, and renaming keys from master (e.g. [`manager.setup.additionalContent.description`](https://github.com/pkp/pkp-lib/blob/master/locale/en_US/manager.xml#L221)) to their ojs-stable-3_1_1 version (e.g. [`manager.setup.additionalContentDescription`](https://github.com/pkp/pkp-lib/blob/ojs-stable-3_1_1/locale/en_US/manager.xml#L216))? Or keep both the master and ojs-stable-3_1_1 versions?

**Comment by mtub** [comment]
@rvdb, we need the translations for master/stable-3_1_2, this is the highest priority (also: upcoming next release). I suggest you complete these PRs and turn your attention to the previous stable branch (`ojs-stable-3_1_1`) after that, aligning the nl_NL files with the en_US files. Maybe you will have to dig up some of the translations you already did for OJS 3.1.1 but deleted for the master branch.

So yes, keep both, but start with master/stable-3_1_2 in the existing PRs and send PRs for the other branch later on.

**Comment by rvdb** [comment]
@mtub I have realized only later that I had branched my `tra-nl_NL` branch in both the ojs and pkp-lib repos from `master`, instead of `stable-3_1_2`. I can imagine the en_US locale files for both branches contain different locale keys, so my translation in this PR might still be incomplete against stable-3_1_2. 

For the plugin submodules, I have done a better job (I guess) by branching off a `tra-nl_NL-3_1_2` from `stable-3_1_2` in my fork, and filing my PRs against that branch.

Should I do the same for the ojs and pkp-lib repos, to make sure the translation is complete for `stable_3_1_2`:

  1. close these PRs for ojs and pkp-lib
  2. redo and push my changes in a new `tra-nl_NL-3_1_2` branch in my fork
  3. open new PRs for these new commits for ojs and pkp-lib

?

**Comment by mtub** [comment]
This should be fine, the master branch and the stable-3_1_2 branch are not that far from each other. I'd rather complete this here for master and then cherry-pick your commits to the stable branches.

**Comment by mtub** [comment]
I will merge these into master and stable-3_1_2, any you can always update the files from there.

Thank you again for this huge contribution!

**Comment by rvdb** [comment]
Ah, meanwhile, I have redone them, filed PRs against `stable-3_1_2` here:

  - https://github.com/pkp/pkp-lib/pull/4521
  - https://github.com/pkp/ojs/pull/2302

Just wanted to make sure everything was in the right place. Sorry about all the Git fuss, I'm learning by doing. I can repeat the exercise for `ojs-stable-3_1_1` later on for ojs, pkp-lib, and the plugin submodules; should I keep referring to https://github.com/pkp/pkp-lib/issues/4503?

**Comment by mtub** [comment]
superseded by #2301

**Comment by mtub** [comment]
If you want to do the work for `ojs-stable-3_1_1`, that would be great. I will probably look at those only after the weekend and will focus on `stable-3_1_2` and `master` today. Thanks!

**Pull request closed without merging**